code/datums/mood_events/generic_negative_events.dm

```diff
@@ -97,7 +97,7 @@
 	timeout = 2 MINUTES
 
 /datum/mood_event/table/add_effects()
-	if(isfelinid(owner)) //Holy snowflake batman!
+	if(isfeline(owner)) //Holy snowflake batman! // SKYRAT EDIT - FELINE TRAITS. Was: isfelinid(owner)
```

Comment:
Eh. Not a big fan of this one to be honest. I already kind of don't really like it for Felinids as a whole, I'd rather not extend that to even more people.